lib/msal-browser/src/broker/BrokerClientApplication.ts

```diff
@@ -112,6 +115,19 @@ export class BrokerClientApplication extends ClientApplication {
         }
     }
 
+    private async brokeredSilentRequest(validMessage: BrokerAuthRequest, clientPort: MessagePort): Promise<void> {
+        try {
+            const response: BrokerAuthenticationResult = (await this.refreshToken(validMessage.request as SilentRequest)) as BrokerAuthenticationResult;
+            const brokerAuthResponse: BrokerAuthResult = new BrokerAuthResult(InteractionType.SILENT, response);
```

Comment:
Need to find some better names here - `BrokerAuthenticationResult` and `BrokerAuthResult` are a bit too similar.

Reply:
`BrokerAuthResponse` should be fine IMO

Reply:
Done